plugins/modules/clickhouse_info.py

```diff
@@ -421,6 +435,90 @@ def get_users(module, client):
     return user_info
 
 
+def get_settings_profiles(module, client):
+    """Get settings profiles.
+
+    Returns a dictionary with profile names as keys.
+    """
+    query = ("SELECT name, id, storage, num_elements, apply_to_all, apply_to_list, "
+             "apply_to_except FROM system.settings_profiles")
+    result = execute_query(module, client, query)
+
+    if result == PRIV_ERR_CODE:
+        return {PRIV_ERR_CODE: "Not enough privileges"}
+
+    profile_info = {}
+    for row in result:
+        profile_info[row[0]] = {
+            "id": str(row[1]),
+            "storage": row[2],
+            "num_elements": row[3],
+            "apply_to_all": row[4],
+            "apply_to_list": row[5],
+            "apply_to_except": row[6],
+        }
+
+    return profile_info
+
+
+def get_quotas(module, client):
+    """Get quotas.
+
+    Returns a dictionary with quota names as keys.
+    """
+    query = ("SELECT name, id, storage, keys, durations, apply_to_all, "
+             "apply_to_list, apply_to_except FROM system.quotas")
+    result = execute_query(module, client, query)
+
+    if result == PRIV_ERR_CODE:
+        return {PRIV_ERR_CODE: "Not enough privileges"}
+
+    quota_info = {}
+    for row in result:
+        quota_info[row[0]] = {
+            "id": str(row[1]),
+            "storage": row[2],
+            "keys": row[3],
+            "durations": row[4],
+            "apply_to_all": row[5],
+            "apply_to_list": row[6],
+            "apply_to_except": row[7],
+        }
+
+    return quota_info
+
+
+def get_server_version(module, client):
+    """Get server version.
```

Comment:
I'm not sure if it's a bug of UI but i see this function now appear twice in the code. Could you please check?